code/game/objects/structures/signs/signs_plaques.dm

```diff
@@ -26,6 +26,23 @@
 	desc = "Next to the extremely long list of names and job titles, there is a drawing of a little child. The child appears to be retarded. Beneath the image, someone has scratched the word \"PACKETS\"."
 	icon_state = "kiddieplaque"
 
+/obj/structure/sign/plaques/ai_password
+	name = "\improper AI default password"
+	desc = "This plaque contains the default password for AI control consoles onboard this station."
+	var/control_code = "BUG"
+
+/obj/structure/sign/plaques/ai_password/Initialize(mapload)
+	. = ..()
+	if(mapload)
```

Comment:
Why only on map load?

Reply:
Doesn’t do much you’re right, if we did somehow have a way to make plaques then I wouldn’t want the new plaque to show the current password, but we don’t so I’m going to remove this